clarity-ext-scripts/deployment/manifest.txt

```diff
@@ -0,0 +1,16 @@
+ConfigSlicerVersion=3.1.14.7
```

Comment:
Should this one be commited?

Reply:
Yeah, I was thinking that. It's like a lock file which says what we want to export. Then we don't have to do this selection step again and again, but we might need to export the package again (similar to specifying requirements in requirements.txt and actually pip installing, we check in the requirements file but not the libraries).